### ISSUE TITLE
Scroll bars for mention list

### ISSUE THREAD

**Issue by felipegiotto** [ckeditor/ckeditor5-mention#81]
Currently, if the mention list has too many items, the suggestion box is very big and goes out of the screen. 

I can use CSS to specify a "max-height" and "overflow-y: scroll" to make it smaller, but then I cannot use the keyboard to select the items, because the focus goes out of the visible part.

I can also limit the number of listed items (e.g. 10), but in this case I need to show all the options to the user.

Suggestion: allow the user to navigate between the items even if there are some CSS properties "max-height / overflow" to limit the visible part.

**Comment by Reinmar** [comment]
> I can use CSS to specify a "max-height" and "overflow-y: scroll" to make it smaller, but then I cannot use the keyboard to select the items, because the focus goes out of the visible part.

Right. This is a bug on our side. If we recommend using `max-height`, we need to handle this.
